01-tax-freedom-day-2016.csv

```diff
@@ -1,14 +1,14 @@
 id,stateAbbr,stateName,taxFreedomDay,taxFreedomDayRank
-0,United States,United States,4/24/2016,
-1,Alabama,Alabama,4/8/2016,4
+0,United States,United States,4/24/2014,
+1,Alabama,Alabama,4/8/2017,4
 2,Alaska,Alaska,4/10/2016,7
 4,Arizona,Arizona,4/15/2016,16
 5,Arkansas,Arkansas,4/13/2016,12
 6,California,California,4/30/2016,45
 8,Colorado,Colorado,4/24/2016,37
 9,Connecticut,Connecticut,5/21/2016,50
 10,Delaware,Delaware,4/18/2016,24
-11,District of Columbia,District of Columbia,4/26/2016,39*
+11,District of Columbia,District of Columbia,4/26/2016,-39
```

Comment:
Excel interprets numbers in parenthesis as negative number (thanks, accountants!), resulting in this ranking error.